src/LdapList.php

```diff
@@ -65,7 +65,7 @@ public function bind()
         }
 
         if ($allFailed) {
-            throw new AuthLDAP_Exception('No bind successfull');
```

Comment:
This wording was used deliberately as the library tries multiple binds and none of these is successful.

Reply:
Ok I misunderstood the context, 'No bind successful' is a viable alternative :)